docs/installation.rst

```diff
@@ -324,3 +324,120 @@ To enable DEBUG on your docker server, call docker run with ``-e GA4GH_DEBUG=Tru
 This will set the environment variable which is read by config.py
 
 You can then get logs from the docker container by running ``docker logs (container)`` e.g. ``docker logs ga4gh_demo``
+
+----------------------------------------------
+Installing the development version on Mac OS X
+----------------------------------------------
+
+**Prerequisites**
+
+First install libraries and header code for
+`Python 2.7 <https://www.python.org/download/releases/2.7/>`_.
+It will be a lot easier if you have `Homebrew <http://brew.sh/index.html>`_,
+the "missing package manager" for OS X, installed first.
+To install Homebrew, paste the following at a Terminal prompt ($):
+
+.. code-block:: bash
+
+  $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+Now use ``brew install`` to install Python if you don't have Python 2.7
+installed and then ``pip install``, which comes with Python, can be used to
+install virtual environment:
+
+.. code-block:: bash
+
+  $ brew install python
+  $ pip install virtualenv
+
+**Install**
+
+Download source code from GitHub to the project target folder, here assumed to 
+be ``~/ga4gh``: (If you haven't already done so, 
+`set up github <https://help.github.com/articles/set-up-git/>`_ 
+to work from your command line.)
+
+.. code-block:: bash
+
+  $ git clone https://github.com/ga4gh/server.git
+
+Before installing Python library dependencies, create a virtualenv sandbox to 
+isolate it from the rest of the system, and then activate it:
+
+.. code-block:: bash
+
+  $ cd server
+  $ virtualenv ga4gh-env
+  $ source ga4gh-env/bin/activate
+
+Install Python dependencies:
+
+.. code-block:: bash
+
+  (ga4gh-env) $ pip install -r dev-requirements.txt
+
+You may encounter the error "ssl.h not found"; if so, you will have to
+``brew install openssl`` after deactivating the virtualenv:
+
+.. code-block:: bash
+
+  (ga4gh-env) $ brew install openssl
+
+If the error persists, you need to tell the compiler about the newly 
+downloaded headers by pasting 
+``export C_INCLUDE_PATH="/usr/local/opt/openssl/include"`` into 
+``~/.profile`` file: 
+
+.. code-block:: bash
+
+  (ga4gh-env) $ vim ~/.profile
+
+Press "i" to switch to INSERT mode, paste
+``export C_INCLUDE_PATH="/usr/local/opt/openssl/include"``, press "esc", 
+type ``:wq`` and press "return (enter)" to save and exit. 
+
+``source`` the edited file to notify the compiler of the change and 
+``echo`` it to verify the change:
+
+.. code-block:: bash
+
+  (ga4gh-env) $ source ~/.profile
+  (ga4gh-env) $ echo $C_INCLUDE_PATH
+  (ga4gh-env) $ pip install -r dev-requirements.txt
+
+**Test and run**
+
+Run tests to verify the install:
+
+.. code-block:: bash
+
+  (ga4gh-env) $ python scripts/run_tests.py
+
+If you deactivated the virtualenv, ``source ga4gh-env/bin/activate``
+to reactivate the ga4gh-env virtualenv.
+ 
+Download and extract the example data:
```

Comment:
Maybe we could put in a link to the tutorial here rather than spell this out again? It's pretty tedious updating these version numbers all the time.